content/en/tests/test_impact_analysis/setup/go.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Test Impact Analysis for Go
+code_lang: go
+type: multi-code-lang
+code_lang_weight: 0
+aliases:
+  - /continuous_integration/intelligent_test_runner/go/
+  - /continuous_integration/intelligent_test_runner/setup/go/
+  - /intelligent_test_runner/setup/go
+further_reading:
+    - link: "/continuous_integration/tests"
+      tag: "Documentation"
+      text: "Explore Test Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI Visibility"
+---
+
+## Compatibility
+
+Test Impact Analysis is only supported on `orchestrion >= 0.9.5 (dd-trace-go >= 1.70.0)`.
+
+## Setup
+
+### Test Optimization
+
+Prior to setting up Test Impact Analysis, set up [Test Optimization for Go][1]. If you are reporting data through the Agent, use v6.40 and later or v7.40 and later.
+
+{{% ci-itr-activation-instructions %}}
+
+## Run tests with Test Impact Analysis enabled
+
+After completing setup, run your tests by using `go test` with the following code coverage options:
+
+```bash
+orchestrion go test ./... -cover -covermode=count -coverpkg ./...
+```
+
+1. `-cover`: The Test Impact Analysis feature uses the built-in Go's code coverage processor, so we need to enable code coverage collection in the `go test` command.
+
+2. `-covermode`: must be either `count` or `atomic`. Because `set` is not supported, setting this value will disable test impact analysis.
```

Comment:
```suggestion
2. `-covermode`: must be either `count` or `atomic`. Because `set` is not supported, setting this value disables test impact analysis.
```